server/src/main/java/org/elasticsearch/script/DynamicMap.java

```diff
@@ -19,28 +19,20 @@
 
 package org.elasticsearch.script;
 
-import org.apache.logging.log4j.LogManager;
-import org.elasticsearch.common.logging.DeprecationLogger;
-
 import java.util.Collection;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Function;
 
-public final class DeprecationMap implements Map<String, Object> {
-
-    private static final DeprecationLogger deprecationLogger =
-        new DeprecationLogger(LogManager.getLogger(DeprecationMap.class));
+public final class DynamicMap implements Map<String, Object> {
```

Comment:
Could you please add a brief javadoc explaining what this class is?

Reply:
Added.